Backend/Underdogs-Struggling/pom.xml

```diff
@@ -40,6 +46,13 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+		
+		<!-- H2 Database -->
+		<dependency>
+				<groupId>com.h2database</groupId>
+				<artifactId>h2</artifactId>
+				<scope>test</scope>
+		</dependency>
```

Comment:
Why we need H2?

Reply:
It is a in-memory database for the unit test scope

Reply:
I believe we have some inconsistency here. 

First of all, we have no plan for unit test or integration test at least for now, so we only need to test locally. Secondly, even for local testing, we should use MySQL database. I have written some dummy data to fill the database for testing purpose. I am not familiar with H2 DB, and I still don't understand why H2 is used here.